src/components/SideMenu/SideMenu.js

```diff
@@ -47,33 +37,33 @@ class SideMenu extends Component {
           </div>
         </span>
 
-        <div className="sidemenu__nav" style={{ width: sidemenu }}>
+        <div className="sidemenu__background" onClick={() => this.closeSideMenu()} style={{ display: background }}></div>
+        <div className="sidemenu__nav" style={{ transform: sidemenu }}>
           <span className="sidemenu__close" onClick={() => this.closeSideMenu()}>&times;</span>
           <div className="sidemenu__list">
-            <div className="sidemenu__droptitle" onClick={() => this.dropDown1()}>
-              <span className="sidemenu--title">FRC Robotics</span><span className="sidemenu--arrow">&#9660;</span>
```

Comment:
@nicholasmiller1 Another nit to pick (or is it knit?): for BEM, the part of the class name after the `--` is the modifier - think a hover state or other temporary modification. I think `sidebar__title` is more appropriate here.